app/js/checkAnswers.js

```diff
@@ -60,31 +60,25 @@ function getUserAnswers(questionAmount) {
     return answers
 }
 
-document.querySelector('#finish').addEventListener('click', function(e) {
-    e.preventDefault()
-    const userAnswers= getUserAnswers(questionAmount)
-
-    checkAnswers(userAnswers).then(function(result) {
-        let percentResult
-        let answered
-
-        if (result.score || result.score === 0) {
-            document.querySelector('#question_page').style.display = 'none'
-            document.querySelector('#result_page').style.display = 'block'
-            percentResult = getPercentResult(result.score, questionAmount)
-            answered = getAnswered(userAnswers, questionAmount)
-            displayResult(result.score, percentResult, answered)
-            handleResponseFromAPI(sendUserResults(result))
-        } else {
-            let body = document.querySelector('body')
-            let html = body.innerHTML
-            html += '<p class="error_message text-danger">Please contact admin. Answers cannot be checked at present.</p>'
-            body.innerHTML = html
-        }
-    })
+document.querySelector('#finish', '#modal-finish').addEventListener('click', function(e) {
```

Comment:
Query selector will only return #finish not #modal-finish - remove #modal-finish